src/Components/StrategyControls/useStrategy.ts

```diff
@@ -93,11 +96,52 @@ REASON: \${Explain why you made this decision}
 \`\`\`
   `;
 
-  const { takeTradingViewScreenshot, setScreenshot } = useScreenshot();
-
-  const runStrategyProd = async () => {
+  // Capture a screenshot
+  const handleBlobs = async (): Promise<Blob[]> => {
```

Comment:
this function is misleading. 
- why an array of Blob ? 
- why "handle" -> getBlob
- it's just a wrapper for "takeTradingViewScreenshot", why not use directly "takeTradingViewScreenshot" ?